pkg/crc/cluster/status.go

```diff
@@ -17,7 +17,7 @@ func WaitForClusterStable(ctx context.Context, ip string, kubeconfigFilePath str
 	startTime := time.Now()
 
 	retryDuration := 30 * time.Second
-	retryCount := 20 // 10 minutes
+	retryCount := 30 // 15 minutes
```

Comment:
10 minutes is already a very long time to wait for failures, I'd rather not add 5 more minutes to it. Maybe make this conditional on proxy usage?

Reply:
Done